bot/botrest/server.go

```diff
@@ -217,15 +204,19 @@ func HandleChannelPermissions(w http.ResponseWriter, r *http.Request) {
 	gId, _ := strconv.ParseInt(pat.Param(r, "guild"), 10, 64)
 	cId, _ := strconv.ParseInt(pat.Param(r, "channel"), 10, 64)
 
-	guild := bot.State.Guild(true, gId)
+	guild := bot.State.GetGuild(gId)
 	if guild == nil {
 		internalapi.ServerError(w, r, errors.New("Guild not found"))
 		return
 	}
 
-	perms, err := guild.MemberPermissions(true, cId, common.BotUser.ID)
-
+	member, err := bot.GetMember(gId, common.BotUser.ID)
 	if err != nil {
+		internalapi.ServerError(w, r, errors.New("Could not find bot member"))
```

Comment:
should there be a `return` after this ?

Reply:
The professional nitpicker is back at his workplace 👀